api/migrations/20180927110244-remove-payments.js

```diff
@@ -0,0 +1,59 @@
+exports.up = async (r, conn) => {
+  const betaSupporterIds = await r
+    .db('spectrum')
+    .table('recurringPayments')
+    .filter({ planId: 'beta-pro' })
+    .map(row => row('userId'))
+    .run(conn)
+    .then(cursor => cursor.toArray());
+
+  const addSupporterFieldToUsersPromises = () =>
+    betaSupporterIds.map(async userId => {
+      return await r
+        .db('spectrum')
+        .table('users')
+        .get(userId)
+        .update({
+          betaSupporter: true,
+        })
```

Comment:
Let's just do this in one go instead of doing n queries? (n = number of beta users)

```JS
const addSupporterFieldToUsersPromise = r.db('spectrum').table('users').getAll(...betaSupporterIds).update({ betaSupporter: true }).run(conn);
```

Reply:
Good call. One sec.